cmd/apprepository-controller/pkg/apis/apprepository/v1alpha1/types.go

```diff
@@ -47,9 +47,11 @@ type AppRepositorySpec struct {
 	DockerRegistrySecrets []string `json:"dockerRegistrySecrets,omitempty"`
 	// In case of an OCI type, the list of repositories is needed
 	// as there is no API for the index
+	// DEPRECATED: Use FilterRules instead
```

Comment:
I guess you're leaving this until all the dependent functionality is implemented (aka not breaking master with this PR), aren't you?

Reply:
yes, at the moment, the filters and the ociRepositories are completely different things, I will merge those in a future PR.